src/vs/workbench/contrib/tasks/electron-browser/task.contribution.ts

```diff
@@ -2749,3 +2751,19 @@ ProblemMatcherRegistry.onMatcherChanged(() => {
 	updateProblemMatchers();
 	jsonRegistry.notifySchemaChanged(schemaId);
 });
+
+const configurationRegistry = Registry.as<IConfigurationRegistry>(Extensions.Configuration);
+configurationRegistry.registerConfiguration({
+	id: 'Tasks',
+	order: 100,
+	title: nls.localize('tasksConfigurationTitle', "Tasks"),
+	type: 'object',
+	properties: {
+		'tasks.terminal.windowsAllowConpty': {
+			markdownDescription: nls.localize('tasks.terminal.windowsAllowConpty', "Works in conjunction with the terminal.integrated.windowsEnableConpty setting. Both must be enabled for tasks to use conpty. Defaults to false."),
```

Comment:
I think using `#terminal.integrated.windowsEnableConpty#` would create a link in the settings editor